.github/workflows/rust.yml

```diff
@@ -65,42 +65,73 @@ jobs:
           # this key equals the ones on `linux-build-lib` for re-use
           key: cargo-cache-benchmark-${{ hashFiles('datafusion/**/Cargo.toml', 'benchmarks/Cargo.toml', 'datafusion-cli/Cargo.toml') }}
 
-      - name: Check workspace without default features
+      - name: Check datafusion without default features
+        # Some of the test binaries require the parquet feature still
```

Comment:
Is including those tests in the `parquet` feature a good idea? I don't know which case needs that. But we would run those tests later with `cargo test`. Ignoring them here should be fine.